logic/.env

```diff
@@ -0,0 +1,2 @@
+TOKEN1={"access_token":"ya29.Il-iB3Mnnv33Sf9CXT4k6scL4wWTKfyx3zf047TY7ySWIzHJ4cuX3R1cl1VtJwn_v8GJUFMIWyoOn7QdAVrVSwN4HEGuOw7txAmlOxIPX-znDDKuhIEQDzFkB3XViC029Q","refresh_token":"1//048aFuJt4M48mCgYIARAAGAQSNgF-L9IrXe94830iT6_9rrm_YH7DtpMN8Djgh6dcx2IHk3OJWqjm-Sh3FUcvScfo8qBW4ZbuGg","scope":"https://www.googleapis.com/auth/calendar","token_type":"Bearer","expiry_date":1571518348287}
```

Comment:
This should be .env.example with a description of what the .env needs, not real data.